release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -57,7 +57,8 @@ cluster infrastructure for you.
 {product-title} requires all machines, including the computer that you run the
 installation process on, to have direct internet access to pull images for
 platform containers and provide telemetry data to Red Hat. You cannot specify a
-proxy server for {product-title}.
+proxy server for {product-title} {product-version}. This functionality will be
+reintroduced in a future release.
```

Comment:
Proxy stuff is going to be 4.2, right?  When this lands in 4.2, `{product-version}` will expand to 4.2 and be wrong.  I expect you want to hard-code 4.1 in these 4.1 release notes.

Reply:
@wking The 4.1 release notes are not part of the 4.2 branch, so the {product-version} won't be bumped to 4.2 in the 4.1 release notes file.